File: CS-Examples/22_TextBoxes/TextBoxWithWrapText.cs
using Spire.Xls;
using Spire.Xls.Core.Spreadsheet.Shapes;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace TextBoxWithWrapText
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            // Create a new workbook object
            Workbook workbook = new Workbook();

            // Load the Excel document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\TextBoxSampleB.xlsx");

            // Get the first worksheet from the workbook
            Worksheet sheet = workbook.Worksheets[0];

            // Get the text box
            XlsTextBoxShape shape = sheet.TextBoxes[0] as XlsTextBoxShape;

            // Set wrap text
            shape.IsWrapText = true;

            // Specify the output filename for the workbook
            string output = "TextBoxWithWrapText.xlsx";

            // Save the modified workbook to a file
            workbook.SaveToFile(output, ExcelVersion.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            // View the document
            FileViewer(output);
        }

        private void FileViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}

File: VB-Examples/22_TextBoxes/TextBoxWithWrapText.vb
Imports Spire.Xls
Imports Spire.Xls.Core.Spreadsheet.Shapes
Imports System.ComponentModel
Imports System.Text

Namespace TextBoxWithWrapText
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new instance of the Workbook class
            Dim workbook As New Workbook()

            ' Load an existing workbook from a file
            workbook.LoadFromFile("..\..\..\..\..\..\Data\TextBoxSampleB.xlsx")

            ' Get the first worksheet from the workbook
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Try to cast the first TextBox shape on the worksheet to XlsTextBoxShape
            Dim shape As XlsTextBoxShape = TryCast(sheet.TextBoxes(0), XlsTextBoxShape)

            ' Set the IsWrapText property of the TextBox shape to true
            shape.IsWrapText = True

            ' Specify the output file name
            Dim output As String = "TextBoxWithWrapText.xlsx"

            ' Save the modified workbook to a file in Excel 2013 format
            workbook.SaveToFile(output, ExcelVersion.Version2013)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'View the document
            FileViewer(output)
		End Sub

		Private Sub FileViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub
	End Class
End Namespace
